README.md

```diff
@@ -1,2 +1,53 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
 # OOP-school-library
-This app allows you to:  Add new students or teachers. Add new books. Save records of who borrowed a given book and when. This project was part of my Microverse journey and it was designed to make me practice Object Oriented Programming while building a app based on a UML class diagram..
+
+This app allows you to: 
+- Add new students or teachers. 
+- Add new books. 
+- Save records of who borrowed a given book and when. 
+
+This project was part of my Microverse journey and it was designed to make me practice Object Oriented Programming while building a app based on a UML class diagram with Ruby.
+
+## Built With
+
+- Ruby
+
+## Getting Started
+
+To run this project follow these simple steps:
+
+- Clone this repo with git, or [download the zip folder](https://github.com/tiagomarin/OOP-school-library/archive/refs/heads/add-classes-Person-Student-Teacher.zip) and extract to your local folders.
+- Run the app.rb file on the terminal and follow the step by step answering the questions.
+
+
+### Prerequisites
+
+- Ruby installed in your computer. ([check this guide if you need to install](https://github.com/microverseinc/curriculum-ruby/blob/main/simple-ruby/articles/ruby_installation_instructions.md))
+- Any code editor like Vscode or Vim.
+
+
+## Authors
+
+👤 **Tiago Lelinski**
+
+- GitHub: [@tiagomarin](https://github.com/tiagomarin)
+- Twitter: [@LelinskiMarin](https://twitter.com/LelinskiMarin)
+- LinkedIn: [Tiago](https://www.linkedin.com/in/tiago-lelinski-marin/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Part of Microverse curriculum
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] OPTIONAL
MIT license link in your ReadMe is broken, kindly endeavor to fix it.

Reply:
Again, thanks for pointing it out!